CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Please make sure you go through this list before submitting a patch.  The rules
 
 - [ ] Did you check your code is formatted correctly? It's easiest to run
 ````
-find src include -name '*.cc' -or -name '*.h' -or -name '*.c'|xargs clang-format -i -style=file
+find src include -name '*.cc' -or -name '*.h' -or -name '*.c'|fgrep -v proc_service.h|xargs clang-format -i -style=file
```

Comment:
After talking with a more license-interested friend I became pretty confused about what the LGPL really requires here.  I didn't know what to write.  So instead I grabbed the FreeBSD header and made the trivial modifications necessary to make it match glibc.  Wondering if that is ok.